pkg/api/pod/util.go

```diff
@@ -748,11 +748,11 @@ func podLifecycleSleepActionInUse(podSpec *api.PodSpec) bool {
 		if c.Lifecycle == nil {
 			return true
 		}
-		if c.Lifecycle.PreStop != nil && c.Lifecycle.PreStop.Sleep != nil {
+		if lc := c.Lifecycle.PreStop; lc != nil && lc.Sleep != nil {
```

Comment:
I'd prefer to revert this change, and the similar changes below. This is a stylistic opinion, not something that is worth refactoring.

Reply:
agree, let's keep this focused on the exact change required for the fix for backporting

Reply:
I think we should make this sort of change, but I'm fine if not in this PR.   We have absolutley seen cut-paste errors which end up with something like:

```
if obj.Sub.OneThing != nil && obj.Sub.OtherThing.Field != nil {
```

Reply:
yeah, +1, in master after we backport the exact fix

Reply:
sure, removed these changes